build-support/travis/travis.yml.mustache

```diff
@@ -168,14 +171,24 @@ py27_linux_test_config: &py27_linux_test_config
   <<: *base_linux_test_config
   stage: *test_cron
   env:
-    - &py27_linux_test_config_env BOOTSTRAPPED_PEX_KEY_SUFFIX=py27.linux
+    - &py27_linux_test_config_env >
+      BOOTSTRAPPED_PEX_KEY_SUFFIX=py27.linux
 
 py36_linux_test_config: &py36_linux_test_config
   <<: *py36_linux_config
   <<: *base_linux_test_config
   stage: *test
   env:
-    - &py36_linux_test_config_env BOOTSTRAPPED_PEX_KEY_SUFFIX=py36.linux
+    - &py36_linux_test_config_env >
+      BOOTSTRAPPED_PEX_KEY_SUFFIX=py36.linux
+
+py36_linux_test_config_cron: &py36_linux_test_config_cron
+  <<: *py36_linux_config
+  <<: *base_linux_test_config
+  stage: *test_cron
+  env:
+  - &py36_linux_test_config_env >
+    BOOTSTRAPPED_PEX_KEY_SUFFIX=py36.linux
```

Comment:
I don't recommend adding this new base entry like this because it increases the number of things we need to keep track of / duplicates things. Instead, I think the cleaner approach is for each relevant shard to extend `py36_linux_test_config` and simply override `stage: *test_cron`, like how we do this with the build engine shards:

https://github.com/pantsbuild/pants/blob/698f9cbb6196cf0fbecb1999fde79ec207ede1c3/build-support/travis/travis.yml.mustache#L827-L828

--

If you want to stick with this base entry, then rename it to `py36_linux_test_config_with_pantsd` and add to the env entry `TRAVIS_AND_PANTS_INTEGRATION_TESTS_SHOULD_HAVE_PANTSD="true"`. Then we could remove `&run_tests_under_pantsd` because it would be inlined, which would be great.

Either of these approaches works well - maybe this second is better.

Reply:
I handn't realized one could override that! Thanks!